pkg/network/encoding/format.go

```diff
@@ -176,6 +177,24 @@ func FormatHTTPStats(httpData map[http.Key]http.RequestStats) map[http.Key]model
 	return aggregationsByKey
 }
 
+// Build the key for the http map based on whether the local or remote side is http.
+func httpKeyFromConn(c network.ConnectionStats) http.Key {
+	// Retrieve translated addresses
+	laddr, lport := nat.GetLocalAddress(c)
+	raddr, rport := nat.GetRemoteAddress(c)
+
+	if http.IsHTTP(int(rport)) {
+		return http.NewKey(laddr, raddr, lport, rport, "")
+	}
+
+	if http.IsHTTP(int(lport)) {
+		// Since HTTP data is always indexed as (client, server), we flip the lookup key
+		return http.NewKey(raddr, laddr, rport, lport, "")
+	}
```

Comment:
For later: could this be changed to use direction?  I can see this code being a problem when we need to support HTTP servers on non standard ports.

Reply:
You're right. I'll make sure to revisit this once we add support for non standard ports.